Dockerfile

```diff
@@ -21,8 +21,10 @@ RUN bundle install
 
 ADD . $APP_HOME
 
-ARG COMPILE_ASSETS=false
-RUN if [ "$COMPILE_ASSETS" = "true" ] ; then bundle exec rails assets:precompile ; fi
+# TODO: GOVUK_ASSET_HOST is required to serve the Flash SWF on https://www.gov.uk/government/history/10-downing-street#take-the-tour
+# Once the flash player has been removed (https://trello.com/c/mCfPlz3z/1089-remove-flash-player) this var can also be removed
+# see: https://github.com/alphagov/whitehall/pull/3663#issuecomment-355626368
+RUN GOVUK_ASSET_HOST=http://assets-origin.dev.gov.uk GOVUK_ASSET_ROOT=http://assets-origin.dev.gov.uk GOVUK_APP_DOMAIN=www.gov.uk RAILS_ENV=production bundle exec rails assets:precompile
```

Comment:
Sorry nitpicking. Can you use the production hosts for consistency `https://assets.publishing.service.gov.uk` so it matches GOVUK_APP_DOMAIN.